ref/rol_Java11.adoc

```diff
@@ -26,7 +26,7 @@ For Windows and Linux systems, you can use either the JDK from Oracle or the Jav
 
 == Java SE 11
 
-There are several Java 11 changes that impact Open Liberty. Between Java 8 and Java 11, several good enhancements such as local variables and the Flow API have been provided. However, there are also a few changes that are likely to cause problems for existing applications.
+There are several Java 11 changes that impact Open Liberty. Between Java 8 and Java 11, several good enhancements such as local variables and the Flow API have been provided. There are also a few changes that are likely to cause problems for existing applications.
 
 In Java 11, several modules (groupings of Java packages) were removed from JDK. With the removal of Java EE and CORBA APIs from the JDK, it is important to note that your applications may have been relying on some these removed packages. However, all of the removed packages are already provided by existing Open Liberty features.
```

Comment:
Suggestion:

In Java SE 11, several modules (groupings of Java packages) were removed from the JDK. With the removal of Java EE and CORBA APIs from the JDK, it is important to note that your applications might have been relying on some of these removed packages. However, all of the removed packages are already provided by existing Open Liberty features.